### DISCUSSION TITLE
Comments on a channel witch a range ofsubsribers

### DISCUSSION THREAD

**Discussion by Pixelssquad** [tegal1337/YOMEN#42] [Ideas]
Hello,
Is it possible to ensure that comments are posted on channels with a certain range of subscribers?
This feature is interesting for me

**Comment by fdciabdul** [comment]
Yes it possible to do that .. i can do that

**Reply by EticalPy** [reply]
Please do it. Is interesting for me to

**Reply by fdciabdul** [reply]
i know it bro .. but i have no idea how to implement to this project , in where and when that function should be input

**Reply by EticalPy** [reply]
creo que seria una condicion simple... Como tu tomas el valor de suscribers en getChannel info, si hay esa condicion (de comentar los videos de los canales con un cierto numero de suscribers),  no haces el comentario si el canal no lo cumple...simplemente pasas a otro video. Es verdad que eso podria dar como resultado mucha tardancia en hacer comentarios, porque no los haria sino solamente cunado cumple esta condicion. Pero si tendria como resultado comentar solo los canales con un cierto numero de suscribers

Otra cosa que me gustaria añadir es no hacer comentarios a los videos que los hemos comentado antes....
